src/principal/service.ts

```diff
@@ -250,6 +255,15 @@ export function recordToModel(user: PrincipalRecord): Principal {
 
 }
 
+export function isIdentityValid(identity: string, type: 'user'): boolean {
+
+  const regex = /^(?:[A-Za-z]+:)?/;
+  if (type === 'user' && regex.test(identity)) {
+    return true;
+  }
+  return false;
```

Comment:
This can be shortened to

```suggestion
  return regex.test(identity);
```

Reply:
:exploding_head: